pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/guide.en-us.md

```diff
@@ -0,0 +1,1278 @@
+2. A [Git](https://git-scm.com/downloads) client, to clone the `OVH Docs` repository.
+3. [Helm](https://www.helms.sh), for managing `TrilioVault Operator` releases and upgrades.
+4. [Kubectl](https://kubernetes.io/docs/tasks/tools), for `Kubernetes` interaction.
+
+**Important note:**
+
+In order for `TrilioVault` to work correctly and to backup your `PVCs`, `OVH Managed Kubernetes Cluster` needs to be configured to support the `Container Storage Interface` (or `CSI`, for short). By default it comes with the driver already installed and configured. You can check using below command:
+
+```shell
+kubectl get storageclass
+```
+
+The output should look similar to (notice the provisioner is [hostpath.csi.k8s.io](https://github.com/kubernetes-csi/csi-driver-host-path)):
+
+```text
+NAME                        PROVISIONER                RECLAIMPOLICY   VOLUMEBINDINGMODE   ALLOWVOLUMEEXPANSION   AGE
+csi-cinder-classic          cinder.csi.openstack.org   Delete          Immediate           true                   3d
+csi-cinder-high-speed       cinder.csi.openstack.org   Delete          Immediate           true                   3d
+csi-hostpath-sc (default)   hostpath.csi.k8s.io        Retain          Immediate           false                  2d
+```
+
+## Step 1 - Installing TrilioVault for Kubernetes
+
+In this step, you will learn how to deploy `TrilioVault` for `OVH Managed Kubernetes Cluster`, and manage `TVK` installations via `Helm`. Backups data will be stored in the `OVH Object Storage` bucket created earlier in the [Prerequisites](#prerequisites) section.
+
+`TrilioVault` application can be installed as below:
+
+- The `TrilioVault Operator` (installable via `Helm`). You define a `TrilioVaultManager` CRD, which tells `TrilioVault` operator how to handle the `installation`, `post-configuration` steps, and future `upgrades` of the `Trilio` application components.
+- The `TrilioVault Manager` via the [k8s-triliovault] yaml definition (https://docs.trilio.io/kubernetes/use-triliovault/installing-triliovault#upstream-kubernetes)
+	
+
+### Installing TrilioVault Operator and Manager Using Helm
+
+**Important note**:
+
+`OVH Docs` tutorial is using the `Cluster` installation type for the `TVK` application (`applicationScope` Helm value is set to `"Cluster"`). All examples from this tutorial rely on this type of installation to function properly.
+
+Please follow the steps below, to install `TrilioVault` via `Helm`:
+
+1. First, clone the `OVH Docs` Git repository and change directory to your local copy:
+
+    ```shell
+    git clone https://github.com/ovh/docs.git
+
+    cd docs/pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/
+    ```
+
+2. Next, add the `TrilioVault` Helm repository, and list the available charts:
```

Comment:
Hello. Knowing from previous experience this list will be a list of ones (1.) when deployed. 
Please use <ol> and <li> for discontinued lists such as this one. See https://www.markdownguide.org/basic-syntax/#ordered-lists for details

Reply:
Thank you @OVH-Team-Guides for this suggestion. I will update the PR with the ordered list method at all required places.